osf/metrics.py

```diff
@@ -269,6 +269,19 @@ def get_latest_user_institution_project_document(cls, user, institution):
 
         return response[0]
 
+    @classmethod
+    def get_current_user_metrics(cls, institution) -> list:
+        """
+        Gets the most recent document for every unique user.
+        :param institution: Institution
+        :return: list
+        """
+        search = cls.filter_institution(institution).sort('timestamp')
+        yesterday = dt.date.today() - dt.timedelta(days=1)
+
+        search = search.filter('range', timestamp={'gte': yesterday})
```

Comment:
I wonder if there's an easy way to get the last set of records, rather than yesterday's records. If, for whatever reason, metrics didn't run one day, it would be nice if it still returned some data. If we can, I'd change what we did in John's query as well.